### PR TITLE
[RHODA][UI]Adding test for adding duplicate provider account name

### PR DESCRIPTION
**Pull request by ikanias**
This PR adds a new test suite which will be used for test cases with generic use.
The test suite contains a test case where a user tries to import a new provider account with a provider name that already exists.
The PR includes the following:
1. A new test suite file: generic_tests.robot
2. An update to the file: resources/keywords/import_provider_account.resource for adding necesary keywords for the test case
3. An update to the file: resources/object_repo/import_dbaasinventory_obj.resource for adding the relevant web element
[generic_tests_test_case.zip](https://github.com/RHODA-lab/rhoda-ci/files/9017086/generic_tests_test_case.zip)

This PR resolves Jira ticket DBAAS-693

### CODE REVIEW

File: tests/generic_tests.robot [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Resource            ../resources/keywords/deploy_application.resource
+
+Suite Setup         Set Library Search Order    SeleniumLibrary
+Suite Teardown      Tear Down The Test Suite
+Test Setup          Given The Browser Is On Openshift Home Screen
+Test Teardown       Close Browser
+
+
+*** Test Cases ***
+Scenario: Verify error message for duplicate provider account friendly name
```
Comment (mrrajan) [inline tests/generic_tests.robot]: @ikanias Please update the test case name with Title Case as "Scenario: Verify Error Message for Duplicate Provider Account Friendly Name"

File: tests/generic_tests.robot [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    [Tags]    smoke   RHOD-84-1
+    When User Imports Valid CockroachDB Provider Account
+    And User Navigates To Import Database Provider Account Screen From Database Access Page
+    And User Enters Data To Import Duplicate CockroachDB Provider Account
+    Then An Error Message Appears For An Existing Provider Account Name
```
Comment (mrrajan) [inline tests/generic_tests.robot]: @ikanias Can we update this to "Duplicate Error Message Appears For Existing Provider Account Name"? LMKWYT

Comment (ikanias) [reply tests/generic_tests.robot]: Hey Rajan,
'Duplicate Error Message Appears...' implies that the error message is displayed twice :-)
How about 'Duplicate Provider Account Name Error Message Appears'?

File: tests/generic_tests.robot (hunk 1 above)
Comment (ikanias) [reply tests/generic_tests.robot]: Done

File: tests/generic_tests.robot (hunk 2 above)
Comment (mrrajan) [reply tests/generic_tests.robot]: Sounds good

**Comment by ikanias** [comment]
[/home/ikanias/git/rhoda-ci/test-output/rhoda-666-2022-06-29-5533/log.html](url)

**Review by harshkumarRH — APPROVED** [review]
Looks good to me!
May need refactoring of Test Suite name and scope, can be discussed later

**Comment by ikanias** [comment]
Closing PR because changes are already merged.
